src/testdrive/src/action/avro_ocf.rs

```diff
@@ -20,11 +20,21 @@ pub struct WriteAction {
     path: String,
     schema: String,
     records: Vec<String>,
+    /// If present, one of the strings understood by [`avro::Codec`]
```

Comment:
This doc comment isn't quite right, fwiw—it's an actual `avro::Codec` by the time it gets here.